### PR TITLE
Updated buffer size

### PR DESCRIPTION
**Pull request by oliverfoster**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-  "name": "image-size",
+  "name": "image-size-big-max-buffer",
```
Comment (shinnn) [inline package.json]: Nope.

File: package.json [hunk 2]
```diff
@@ -14,7 +14,7 @@
   },
   "repository": {
     "type": "git",
-    "url": "git@github.com:netroy/image-size.git"
+    "url": "git@github.com:oliverfoster/image-size-big-max-buffer.git"
```
Comment (shinnn) [inline package.json]: Nope.

**Comment by oliverfoster** [comment]
Apologies dude, i forgot that subsequent changes add themselves to previous pr's. I wasn't expecting you to respond so quickly

**Comment by oliverfoster** [comment]
Is the proposed change ok though? adding * 100 to the buffer size?

**Comment by LinusU** [comment]
@oliverfoster Why do you need a bigger buffer? What is the problem that you are trying to solve?

**Comment by oliverfoster** [comment]
https://github.com/image-size/image-size/issues/34 < this one. I got this on larger jpegs.

**Pull request closed without merging**